instrumentation/opentelemetry-instrumentation-aws-lambda/tests/test_aws_lambda_instrumentation_manual.py

```diff
@@ -440,35 +552,105 @@ def test_api_gateway_http_api_proxy_event_sets_attributes(self):
             },
         )
 
-    def test_lambda_handles_list_event(self):
+    def test_alb_conventional_event_sets_attributes(self):
         AwsLambdaInstrumentor().instrument()
 
-        mock_execute_lambda([{"message": "test"}])
+        mock_execute_lambda(MOCK_LAMBDA_ALB_EVENT)
 
-        spans = self.memory_exporter.get_finished_spans()
+        span = self.memory_exporter.get_finished_spans()[0]
```

Comment:
Wdyt convert this logic into a function that get finished spans, asserts the len based on a num_spans?

Reply:
I won't add other wrappers, you can make it a bit better with unpacking the sequence if you want:

```
        span, = self.memory_exporter.get_finished_spans()
```

Reply:
I've tweaked the logic a little to first ensure we have the number of spans we expect, and then grab the first element.